CONTRIBUTING.md

```diff
@@ -10,17 +10,19 @@ Machine is a part of the [Docker](https://www.docker.com) project, and follows
 the same rules and principles. If you're already familiar with the way
 Docker does things, you'll feel right at home.
 
-Otherwise, go read
-[Docker's contributions guidelines](https://github.com/docker/docker/blob/master/CONTRIBUTING.md).
+Otherwise, please read [Docker's contributions
+guidelines](https://github.com/docker/docker/blob/master/CONTRIBUTING.md).
+
+# Building
 
 The requirements to build Machine are:
 
 1. A running instance of Docker
-2. The `bash` shell
```

Comment:
Not sure this should be redacted - `bash` is still required (esp. important for systems where it isn't installed by default).